service/installer/helm/metrics.go

```diff
@@ -35,11 +37,21 @@ var (
 		},
 		[]string{"name"},
 	)
+	helmReleaseFailure = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Namespace: prometheusNamespace,
+			Subsystem: prometheusSubsystem,
+			Name:      "helm_release_failure",
```

Comment:
```suggestion
			Name:      "helm_release_failed",
```

I'm not sure about the metric name. WDYT to this?

Reply:
Not bad, going with your suggestion.